libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1751,6 +1748,61 @@ def generate_and_compare_both_version(input_installing_version, installed_versio
             LOGGER.info("Installing version is not higher than installed version.")
         return installing_version
 
+    def update_sol_for_granular_deploy(self, file_path: str, host_list: list,master_node_list,
+                                       image: str, deployment_type: str, **kwargs):
+        """
+        Helper function to update image in solution.yaml.
+        :param: file_path: Filename with complete path
+        :param: host_list: List of setup hosts
+        :param: image: Image to be used for deployment
+        :param: deployment_type: Type of deployment(Standard/Data-Only)
+        :return: True/False and local file
+        """
+        cvg_count = kwargs.get("cvg_count", 2)
+        data_disk_per_cvg = kwargs.get("data_disk_per_cvg", 0)
```

Comment:
default 0 is allowed for deployment?

Reply:
In prov_k8_cortx_deploy.py class there is one method update_cvg_sol_file so from there i reffered it and make logic according to my data disk per cvg.

Reply:
@pragamj is 0 value for data disk is allowed?

Reply:
if its 0 it will pick all the available disk in the node.